iothub/device/src/Microsoft.Azure.Devices.Client.csproj

```diff
@@ -22,7 +22,7 @@
   </PropertyGroup>
 
   <PropertyGroup>
-    <Version>1.29.0-preview-003</Version>
+    <Version>1.29.0-preview-004</Version>
     <Title>Microsoft Azure IoT Device Client SDK</Title>
```

Comment:
Don't we do this during release?

Reply:
Yes, but since this is the only change going into this release, I included it within the same PR